src/main/java/com/hms_networks/americas/sc/extensions/time/TimeLibConstants.java

```diff
@@ -8,35 +8,18 @@
  */
 public class TimeLibConstants {
 
-  /**
-   * Default username of Ewon Flexy devices. Used as the default username in time utilities unless
-   * otherwise specified by the user.
-   */
-  public static final String DEFAULT_DEVICE_USERNAME = "adm";
-
-  /**
-   * Default password of Ewon Flexy devices. Used as the default password in time utilities unless
-   * otherwise specified by the user.
-   */
-  public static final String DEFAULT_DEVICE_PASSWORD = "adm";
-
-  /** Loopback IP address for accessing the local host. */
-  public static final String LOOPBACK_IP_ADDRESS = "127.0.0.1";
-
   /** Date format generated by the local time offset HTML file in result files. */
   public static final String TIME_OFFSET_DATE_FORMAT = "dd/MM/yyyy HH:mm:ss";
 
   /** Folder where time offset files should be stored on the filesystem. */
   public static final String TIME_OFFSET_DIRECTORY_PATH = "/usr/TimeOffset/";
 
-  /** Name of the time offset file which stores the local time result. */
-  public static final String TIME_OFFSET_RESULT_FILE_NAME =
-      TIME_OFFSET_DIRECTORY_PATH + "TimeOffsetValue.txt";
-
-  /** Name of the time offset file which stores the HTML that generates the local time result. */
-  public static final String TIME_OFFSET_HTML_FILE_NAME =
-      TIME_OFFSET_DIRECTORY_PATH + "TimeOffsetCalculator.shtm";
+  /** Name of the file which stores the exported local time result. */
+  public static final String TIME_OFFSET_LOCAL_TIME_FILE_NAME = "ExportedLocalTimeValue.txt";
 
-  /** Contents of the local time offset HTML file. */
-  public static final String TIME_OFFSET_HTML_FILE_CONTENTS = "<%#ExeSSI,PRINT #0,TIME$%>";
+  /**
+   * The export block descriptor value prefix (file name is appended) used to retrieve the local
+   * time.
+   */
+  public static final String TIME_OFFSET_LOCAL_TIME_EBD_PREFIX = "$dtSE$se\"Time$\"$ftT$fn";
```

Comment:
It might be nice to see a little more documentation on the EBD string. These probably really look like magic to a customer. Even just a note to where its documented in a specific version of a reference guide would be helpful.